spec/views/events/new.html.erb_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
```

Comment:
can delete this file I think - I don't believe that view specs are giving us any value